### PR TITLE
added tracking of main account balance to token handler

### PR DESCRIPTION
**Pull request by AndyGura**

### CODE REVIEW

File: src/TokenHandler.mo [hunk 1]
```diff
@@ -349,6 +351,12 @@ module TokenHandler {
     /// retrieve the sum of all successful consolidations
     public func totalConsolidated() : Nat = totalConsolidated_;
 
+    /// retrieve the sum of all deductions from main account of the token handler
+    public func totalWithdrawnFromMain() : Nat = totalWithdrawnFromMain_;
```
Comment (timohanke) [inline src/TokenHandler.mo:355]: Let's shorten it to `totalWithdrawn`

File: src/TokenHandler.mo [hunk 2]
```diff
@@ -349,6 +351,12 @@ module TokenHandler {
...
+
+    /// retrieve the calculated balance of main account of the token handler
+    public func handlerMainAccountFunds() : Nat = totalConsolidated_ - totalWithdrawnFromMain_;
```
Comment (timohanke) [inline src/TokenHandler.mo:358]: Let's shorten it to `consolidatedFunds()`

**Review by timohanke — COMMENTED** [review]
Just change function names

File: src/TokenHandler.mo (hunk 1 above)
Comment (AndyGura) [reply src/TokenHandler.mo]: Done

File: src/TokenHandler.mo (hunk 2 above)
Comment (AndyGura) [reply src/TokenHandler.mo]: Done

**Review by timohanke — APPROVED** [review]

**Pull request merged**